lib/i18n/tokenizer/base.js

```diff
@@ -470,4 +470,9 @@ module.exports = class BaseTokenizer {
 
         return { tokens, rawTokens, entities };
     }
+
+    detokenize(tokens) {
+        return tokens.join(' ');
+    }
```

Comment:
I'm surprised this works, because there is already a "detokenize" function in this class.